### DISCUSSION TITLE
turbo deployment does not catch errors on build

### DISCUSSION THREAD

**Discussion by kesar** [vercel/community#358] [Help] [answered]
I am using turbo and trying to deploy on vercel the ui (nextjs)

command is: `cd ../.. && npx turbo run build --scope=@everipedia/frontend --includeDependencies --no-deps`

so far when a build fails it keeps the process in yellow hanging till 45min. reached

any idea / suggestion?

**Comment by samcx** [comment, marked as answer]
@kesar Apologies for the late response! Are you still running into build issues?

**Reply by kesar** [reply]
nope, I went back to split repos after so much troubles w monorepos and so little benefits

**Reply by samcx** [reply]
I see. Apologies for the poor experience!

I believe the issue may have been with your command `cd ../.. && npx turbo run build --scope=@everipedia/frontend --includeDependencies --no-deps`. Was this `@everipedia/frontend` the exact name that shows for your sub-directory package.json?

e.g.

![CleanShot 2022-03-11 at 00 44 53@2x](https://user-images.githubusercontent.com/28912696/157833190-b5cb2fa1-41c1-4e0b-9181-0b8ed7b8f783.jpg)
